labeled_data_tracking.csv

```diff
@@ -1,7 +1,5 @@
 CIK,Filename,Initials,Notes
 107815,edgar/data/107815/0000107815-17-000106.txt,KL
-103682,edgar/data/103682/0001193125-16-480850.txt,KL
```

Comment:
A few of these Ex. 21 docs are too long to be rendered in Label Studio when all pages are combined into one image, so it was easier to just exclude them from the labeled dataset.